src/components/table/table_header.tsx

```diff
@@ -20,14 +20,23 @@
 import React, { FunctionComponent } from 'react';
 import { CommonProps } from '../common';
 
-export const EuiTableHeader: FunctionComponent<CommonProps> = ({
+type Props = CommonProps & {
+  withInbuiltTag?: boolean;
+};
```

Comment:
I think we can name this prop a little better and add a prop comment to fill out the props table. Also, since there's more than just `CommonProps`, we'll want to export the TS type by also naming it more specific to this component.

```suggestion
export type EuiTableHeaderProps = CommonProps & {
  /**
   * Automatically adds a wrapping `<tr>` element around the children
   */
  wrapWithTableRow?: boolean;
};
```

Reply:
Then you'll need to update the rest of the file to match